fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -1302,6 +1302,9 @@ public static TPersistentIndexType analyzePersistentIndexType(Map<String, String
             properties.remove(PROPERTIES_PERSISTENT_INDEX_TYPE);
             if (type.equalsIgnoreCase("LOCAL")) {
                 return TPersistentIndexType.LOCAL;
+            } else if (type.equalsIgnoreCase("CLOUD_NATIVE")) {
```

Comment:
If `CLOUD_NATIVE` is specified when creating a table, better to report an error.

Reply:
done